{"name": "test_first", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 == 20", "trace": "def test_first():\n        a = 10\n        b = 20\n    \n>       assert a == b\nE       assert 10 == 20\n\nSeleniumProgram\\test_first_program.py:5: AssertionError"}, "start": 1673445930239, "stop": 1673445930239, "uuid": "c88ce89b-95c0-4d43-ae92-ce5627bc1f77", "historyId": "54701c13c5f1bf9e48a35aaad587bd9f", "testCaseId": "bc64b7aa61bef38ed465753d8f157431", "fullName": "test_first_program#test_first", "labels": [{"name": "suite", "value": "test_first_program"}, {"name": "host", "value": "HP"}, {"name": "thread", "value": "45852-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_first_program"}]}